utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -25,21 +34,20 @@ class UrlImageParser(
 
   val GCS_PREFIX: String = "https://storage.googleapis.com/"
   val GCS_RESOURCE_BUCKET_NAME = "oppiaserver-resources/"
-  var IMAGE_DOWNLOAD_URL_TEMPLATE = "/<entity_type>/<entity_id>/assets/image/<filename>"
+  var IMAGE_DOWNLOAD_URL_TEMPLATE = "<entity_type>/<entity_id>/assets/image/<filename>"
 
-  var targets: ArrayList<BitmapTarget>? = null
+  // TODO below implementation is to load image using Glide Library
```

Comment:
What are the next steps to resolve this TODO? TODOs should always have issues corresponding to them, and should have a clear actionable work item that someone else with less context could take on and resolve.

Is this still an issue that needs to be solved or can we remove this? What additional work needs to be done that this issue corresponds to?

Reply:
There is no any issue in this. I just showed two approaches over here. Using Glide library and Asynctask.  I will keep Glide and remove other code